package/package.sh

```diff
@@ -206,10 +210,13 @@ function dump_syms {
 }
 
 # The main
-build $version $enablesanitizer $static_sanitizer $build_type $branch
-rm -rf ${build_dir}/src
+build $version $enablesanitizer $static_sanitizer $build_type $branch "OFF" "/usr/local/nebula"
 package $strip_enable
 if [[ $dump_symbols == ON ]]; then
     echo ">>> start dump symbols <<<"
     dump_syms
 fi
+
+# tar package
+build $version $enablesanitizer $static_sanitizer $build_type $branch "ON" "/"
```

Comment:
install the tar package into root directory by default?

Reply:
'/' means root of tar package here.